### DISCUSSION TITLE
Connected, waiting for the image

### DISCUSSION THREAD

**Discussion by kytril87** [rustdesk/rustdesk#9549] [Q&A]
Hello,
I need some advice. When I try to connect to the device, I connect normally, but the message Connected, waiting for the image pops up. And this state lasts for several minutes and nothing happens. Does anyone have experience with this?
Thanks a lot.

**Comment by rustdesk** [comment]
which version on each side？ use latest version first, then try out different codec please 

https://github.com/rustdesk/rustdesik/discussions/8107#discussioncomment-9507636

**Comment by kytril87** [comment]
I'm using version 1.2.3 on both devices. I have this version on other devices and some of them run without problems and some of them have this problem

**Reply by 21pages** [reply]
Could you try out https://github.com/21pages/rustdesk/releases/download/directx/rustdesk-1.3.5-x86_64.exe on both side? If waiting for image still happens, paste the log on both sides, remember to remove lines containing personal informations. https://github.com/rustdesk/rustdesk/wiki/FAQ#access-logs
